FILENAME: 29test_reqctx.py, LINE: 46 (Empty exception detected)
FILENAME: 30test_blueprints.py, LINE: 358 (Empty exception detected)
FILENAME: 30test_blueprints.py, LINE: 367 (Empty exception detected)
FILENAME: 36test_appctx.py, LINE: 75 (Empty exception detected)
FILENAME: 36test_appctx.py, LINE: 217 (Empty exception detected)
FILENAME: 38test_testing.py, LINE: 239 (Empty exception detected)
FILENAME: 63__init__.py, LINE: 26 (Empty exception detected)
